test/dao/operational.ts

```diff
@@ -24,4 +27,34 @@ describe('DAO operational Tests', () => {
         expect(await ssvNetworkContract.owner()).equals(helpers.DB.owners[4].address);
     });
 
+    it('Get the network validators count when updating it', async () => {
```

Comment:
let please add a scenario with a removed validator and a liquidated validator where we do a count after each